x-pack/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Elastic-License",
   "scripts": {
     "kbn": "node ../scripts/kbn",
-    "kbn:bootstrap": "rm -rf ../built_assets/canvasStorybookDLL",
+    "kbn:bootstrap": "node plugins/canvas/scripts/clean_storybook_dll",
```

Comment:
thoughts on less specific for kbn:bootstrap?  feels like the file should be named kbn_bootstrap or something.

Reply:
I figure we can keep the script specific to storybook until there is another bootstrap task needed. Since we are also planning on splitting up the plugins, this should make it more clear as to where this should go.